src/libAtomVM/globalcontext.h

```diff
@@ -139,6 +139,19 @@ GlobalContext *globalcontext_new();
  */
 void globalcontext_destroy(GlobalContext *glb);
 
+/**
+ * @brief Gets a Context from the process table, acquiring a lock on the process
```

Comment:
I'm not sure about this, aren't we doing quite the opposite here?

Reply:
Yes! I forgot the very important word **without** acquiring a lock... as described in the details.